paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -93,6 +106,22 @@ public class PaparazziPlugin @Inject constructor(
     }
   }
 
+  private fun Project.configureRootProject() {
+    val resolver = createResolver()
+    tasks.register("generateSnapshotBrowser", GenerateBrowserTask::class.java) { task ->
+      val paths = resolver.map { c -> c.incoming.artifactView { it.lenient(true) }.artifacts.artifactFiles }
+      task.projectRoot.set(providerFactory.provider { layout.projectDirectory.asFile.path })
+      task.snapshotMetadataReports.setFrom(paths)
+      task.output.set(project.layout.buildDirectory.dir("paparazzi"))
+    }
+
+    allprojects.forEach { p ->
+      dependencies.let { d ->
+        d.add(SNAPSHOT_METADATA.declarableName, d.project(mapOf("path" to p.path)))
```

Comment:
If the `path` is all that's needed, would `p.isolated.path` work? I think that should be project isolation compliant.

Reply:
good point, will explore and follow-up